File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        int min = (int)2e9;
        int max = (int)-2e9;
        String answer = "";
        for(int i = 0; i < s.length(); i++){
            String curNum = "";
            if(s.charAt(i) == ' ') continue;
            while(i < s.length() && s.charAt(i) != ' '){
                curNum += s.charAt(i);
                i++;
            }
            int num = Integer.parseInt(curNum);
            if(min > num) min = num;
            if(max < num) max = num;
        }
        answer = (min + " " + max);
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    int min = 2e9;
    int max = -2e9;
    string answer = "";
    for(int i = 0; i < s.length(); i++){
        string curNum;
        if(s[i] == ' ') continue;
        while(s[i] != ' '){
            curNum += s[i];
            i++;
        }
        int num = stoi(curNum);
        if(min > num) min = num;
        if(max < num) max = num;
    }
    answer = (to_string(min) + ' ' + to_string(max));
    return answer;
}